dist/windows/installer-translations/portugueseBR.nsi

```diff
@@ -15,7 +15,7 @@ LangString inst_magnet ${LANG_PORTUGUESEBR} "Abrir links magnéticos com qBittor
 ;LangString inst_firewall ${LANG_ENGLISH} "Add Windows Firewall rule"
 LangString inst_firewall ${LANG_PORTUGUESEBR} "Adicionar regra no firewall do Windows"
 ;LangString inst_pathlimit ${LANG_ENGLISH} "Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)"
-LangString inst_pathlimit ${LANG_PORTUGUESEBR} "Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)"
+LangString inst_pathlimit ${LANG_PORTUGUESEBR} "Desativar limite de caracteres em caminhos do Windows (limite de 260 caracteres, requer Windows 10 1067 ou superior)"
```

Comment:
Apologies for yet another change, but don't you think this would be even better? It also mentions `MAX_PATH` like the original.

`Desativar limite MAX_PATH de 260 caracteres em caminhos do Windows (requer Windows 10 1067 ou superior)`

The rest looks good :+1:

Reply:
I didn't add that because the user wouldn't understand what MAX_PATH is. But sure.